app/views/includes/foot.jade

```diff
@@ -39,4 +39,4 @@ script(type='text/javascript', src='/js/init.js')
 script(type='text/javascript', src='/socket.io/socket.io.js')
 
 //Livereload script rendered
-//script(type='text/javascript', src='http://localhost:35729/livereload.js')
\ No newline at end of file
+//script(type='text/javascript', src='https://localhost:35729/livereload.js')
```

Comment:
Add a new line here.  (to resolve the red sign at the end of the line). All else appears to be well